helm-cli.js

```diff
@@ -233,18 +254,23 @@ ${parametersWithEnvironmentalVariablesArray.join(" ")}`;
 
   logToActivityLog(`Executing ${completeCommand}`);
 
-  return exec(completeCommand, {
+  const result = await exec(completeCommand, {
     env: shellEnvironmentalVariables,
   });
+
+  return {
+    stderr: result.stderr,
+    stdout: redactTokenValue(result.stdout),
+  };
 }
 
-function wrapChartVolumeDefinition(volumeDefinition, chartDirectory) {
-  const [, chartDirectoryName] = splitDirectory(chartDirectory);
+function addExtraPathToVolumeDefinition(volumeDefinition, workingDirectory) {
+  const [, workingDirectoryName] = splitDirectory(workingDirectory);
 
   /* eslint-disable no-param-reassign,no-unused-expressions */
   volumeDefinition.mountPoint.value.endsWith("/")
```

Comment:
Wouldn't this be cleaner to just use `if / else`? Or if you want to join two path segments, you can use `join()` function from `path` node module.

Reply:
I don't understand.

My point was to modify the existing `mountPoint.value` by appending it with the working directory's name. I've replaced the old call with direct `path.basename` call.

Reply:
I think the lines highlighting may be a bit misleading, I've meant to address lines 271-273.